shared/js/ui/views/site.es6.js

```diff
@@ -121,8 +137,41 @@ Site.prototype = window.$.extend({},
                 return
             }
 
-            let url = encodeURIComponent(this.model.tab.url)
-            browserUIWrapper.openExtensionPage(`/html/feedback.html?broken=1&url=${url}`)
+            this.showBreakageForm('reportBrokenSite')
+        },
+
+        _onConfirmBrokenClick: function () {
+            const isHiddenClass = 'is-hidden'
+            this.$managewhitelistli.removeClass(isHiddenClass)
+            this.$confirmbreakageli.addClass(isHiddenClass)
+            this.$body.removeClass('confirmation-active')
+            this.showBreakageForm('toggle')
+        },
+
+        _onConfirmNotBrokenClick: function () {
+            const isTransparentClass = 'is-transparent'
+            this.$confirmbreakagemessage.removeClass(isTransparentClass)
+            this.$confirmbreakage.addClass(isTransparentClass)
+            this.$body.removeClass('confirmation-active')
+            this.closePopupAndReload(1500)
+        },
+
+        _showBreakageConfirmation: function () {
+            this.$body.addClass('confirmation-active')
```

Comment:
Adding a class here so that we can prevent rerenders when the confirmation form is active.